Toolchain/BuildIt.sh

```diff
@@ -78,7 +78,11 @@ GCC_BASE_URL="http://ftp.gnu.org/gnu/gcc"
 buildstep() {
     NAME=$1
     shift
-    "$@" 2>&1 | sed 's|^|\x1b[34m['"${NAME}"']\x1b[39m |'
+    if [ "$(uname -s)" = "Darwin" ]; then
+        "$@" 2>&1 | sed 's|^|['"${NAME}"'] |'
+    else
+        "$@" 2>&1 | sed 's|^|\x1b[34m['"${NAME}"']\x1b[39m |'
+    fi
```

Comment:
```suggestion
    "$@" 2>&1 | sed $'s|^|\x1b[34m['"${NAME}"$']\x1b[39m |'
```

@gunnarbeutner: I don't have a mac to test this, but wouldn't this work on both mac and linux?